kiosk/bin/kiosk-browser

```diff
@@ -6,6 +6,7 @@ import os
 import kiosk_browser
 
 default_toggle_settings_key = 'CTRL+SHIFT+F12'
+default_max_cache_size = 1024*1024*100 # 100MB
```

Comment:
Shall we put this a bit higher? This may in principle mean game assets need to be evicted from cache during dev use when cache limit is not explicitly set (though unlikely that someone loads the majority of assets), which might alter perceived site performance.

Reply:
OK, let's set this low, while possibly ramping up production cap.